build.xml

```diff
@@ -277,6 +277,7 @@ TODO:
       <property name="junit.version" value="4.12"/>
       <artifact:dependencies pathId="junit.classpath" filesetId="junit.fileset">
         <dependency groupId="junit" artifactId="junit" version="${junit.version}"/>
+        <dependency groupId="org.openjdk.jol" artifactId="jol-core" version="0.5"/>
```

Comment:
you'll probably need to add this to `build.sbt` too - did you try running `sbt test`? 

also cc @adriaanm for adding a (test) dependency to scala.